### PR TITLE
WIP support for text file and added link to google fonts

### PR DESCRIPTION
**Pull request by abhishrijoshi**
Added links to google font for new users to choice with eaise .

### CODE REVIEW

**Comment by abhishrijoshi** [comment]
Can anyone please look into this ? 
the image doesnt show where it should be and the text in file if not formatted correctly gets cut,

the download is working though

**Comment by abhishrijoshi** [comment]
Check here https://abhishrijoshi.github.io/text-to-handwriting/

File: index.html [hunk 1]
```diff
@@ -150,34 +150,30 @@ <h1>Text to Handwriting</h1>
...
+			   document.getElementById('inputfile') 
+			   .addEventListener('change', function() { 
+			    var fr=new FileReader(); 
+			     fr.onload=function(){ 
+			    document.getElementById('output') 
+			   .textContent=fr.result; 
+			     } 
+			   fr.readAsText(this.files[0]); 
+			    }) 
+			   </script>
```
Comment (saurabhdaware) [inline index.html:175]: You can move this JavaScript to the JavaScript file and have the input HTML element as it was before. Then in the onload, you can replace the existing lorem ipsum text with the new text.

**Comment by saurabhdaware** [comment]
I am about to archive this repository so I will have to close this. An alternative to this is copying content from the file which shouldn't be that time-consuming. You can definitely maintain the feature in your fork.

**Pull request closed without merging**